### DISCUSSION TITLE
Unable to Install Prisma for a NodeJS applicaiton on Windows

### DISCUSSION THREAD

**Discussion by luthrajat** [prisma/prisma#24250] [Q&A]
### Question

# npm i -D prisma

C:\Users\Administrator\Desktop\v2\next>npm i -D prisma
<Same Error>
C:\Users\Administrator\Desktop\v2\next>npm i -D prisma@5.8.1
npm error code 1
npm error path C:\Users\Administrator\Desktop\v2\next\node_modules\@prisma\engines
npm error command failed
npm error command C:\Windows\system32\cmd.exe /d /s /c node scripts/postinstall.js
npm error node:fs:2707
npm error       const stats = binding.lstat(baseLong, true, undefined, true /* throwIfNoEntry */);
npm error                             ^
npm error
npm error Error: ENOENT: no such file or directory, lstat 'C:\Users\Administrator\AppData\Local\Temp;C:'
npm error     at Object.realpathSync (node:fs:2707:29)
npm error     at ../../node_modules/.pnpm/temp-dir@2.0.0/node_modules/temp-dir/index.js (C:\Users\Administrator\Desktop\v2\next\node_modules\@prisma\get-platform\dist\chunk-LVABMATQ.js:5101:20)
npm error     at __require2 (C:\Users\Administrator\Desktop\v2\next\node_modules\@prisma\get-platform\dist\chunk-UPBZT3NW.js:46:51)
npm error     at ../../node_modules/.pnpm/tempy@1.0.1/node_modules/tempy/index.js (C:\Users\Administrator\Desktop\v2\next\node_modules\@prisma\get-platform\dist\chunk-LVABMATQ.js:14899:19)
npm error     at __require2 (C:\Users\Administrator\Desktop\v2\next\node_modules\@prisma\get-platform\dist\chunk-UPBZT3NW.js:46:51)
npm error     at Object.<anonymous> (C:\Users\Administrator\Desktop\v2\next\node_modules\@prisma\get-platform\dist\chunk-LVABMATQ.js:14957:55)
npm error     at Module._compile (node:internal/modules/cjs/loader:1358:14)
npm error     at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
npm error     at Module.load (node:internal/modules/cjs/loader:1208:32)
npm error     at Module._load (node:internal/modules/cjs/loader:1024:12) {
npm error   errno: -4058,
npm error   code: 'ENOENT',
npm error   syscall: 'lstat',
npm error   path: 'C:\\Users\\Administrator\\AppData\\Local\\Temp;C:'
npm error }
npm error
npm error Node.js v20.13.1
npm error A complete log of this run can be found in: C:\Users\Administrator\AppData\Local\npm-cache\_logs\2024-05-22T03_13_50_371Z-debug-0.log

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it
PostgreSQL

**Comment by ludralph** [comment]
Hi @luthrajat 👋 

The npm error `"ENOENT: no such file or directory, lstat 'C:\Users\Administrator\AppData\Local\Temp;C:'"` suggest that npm is unable to locate that specific directory, likely due to incorrect path construction. The path specified in the error message `(C:\Users\Administrator\AppData\Local\Temp;C:)` seems to be combining the temporary directory with the root drive, which is likely not intended. Your paths should look like 
```
TEMP: C:\Users\Administrator\AppData\Local\Temp
TMP: C:\Users\Administrator\AppData\Local\Temp
```
It should not have any semicolons (`;`) or additional paths. You would need to set the path correctly before trying to install Prisma.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
